data-prepper-plugins/geoip-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/LocationData.java

```diff
@@ -1,43 +1,164 @@
 package com.amazon.dataprepper.plugins.prepper.geoip.provider;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-// This is a first pass of LocationData, simply to make sure lookup correctly passes through data.
-//TODO Find the best way to attach the location data to a document, and modify this class to allow that.
-
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Objects;
 
 public class LocationData {
+    private final String CITY_FIELD = "geo.city_name";
```

Comment:
CITY_NAME_FIELD? Asking because I see things like
* COUNTRY_**ISO**
* REGION_**CODE**

so maybe NAME should be called out too?